app_legacy/Site/Models/User.php

```diff
@@ -88,6 +90,26 @@ protected function getHashIdAttribute(): int
         return app(Optimus::class)->encode($this->getAttribute('ID'));
     }
 
+    /**
+     * Return unlocks separated by unlock mode; both softcore and hardcore in "raw" form
+     *
+     * @return HasMany<PlayerAchievement>
+     */
+    public function playerAchievements(): HasMany
+    {
+        return $this->hasMany(PlayerAchievement::class, 'User', 'User');
```

Comment:
Heads up: this will most likely be replaced by the `ActsAsPlayer` trait as soon as the legacy merge is done in V3. I just thought if that isn't already in there but remembered that we still have two different user models in V2 😅